### PR TITLE
Use only WS codes

### PR DESCRIPTION
**Pull request by mirpo**

### CODE REVIEW

File: src/connection/WsEndpoint.js [hunk 1]
```diff
@@ -163,9 +163,7 @@ class WsEndpoint extends EventEmitter {
             } catch (e) {
                 this.metrics.inc('send:failed')
                 console.error('sending to %s failed because of %s, readyState is', recipientAddress, e, ws.readyState)
-                if (ws.readyState === 2 || ws.readyState === 3) {
-                    ws.terminate()
-                }
+                ws.terminate()
```
Comment (harbu) [inline src/connection/WsEndpoint.js:166]: I think we may want to be careful about terminating a "connecting" socket? Maybe leave `if(ws.readyState !== ws.CONNECTING)`?

File: src/connection/WsEndpoint.js [hunk 2]
```diff
@@ -202,9 +200,7 @@ class WsEndpoint extends EventEmitter {
                 } catch (e) {
                     this.metrics.inc('send:failed')
                     console.error('sending to %s failed because of %s, readyState is', recipientAddress, e, ws.readyState)
-                    if (ws.readyState === 2 || ws.readyState === 3) {
-                        ws.terminate()
-                    }
+                    ws.terminate()
```
Comment (harbu) [inline src/connection/WsEndpoint.js:203]: Same as above.

**Review by harbu — COMMENTED** [review]
LGTM; one question about readyState "connecting" when sending

File: src/connection/WsEndpoint.js (hunk 1 above)
Comment (mirpo) [reply src/connection/WsEndpoint.js:166]: No, we add `ws` to connections only after 'open' or 'connection' event. So ws.readState **must** be => ws.OPEN

Comment (harbu) [reply src/connection/WsEndpoint.js:166]: Ah yes, indeed.

**Review by harbu — APPROVED** [review]

**Pull request merged**